### PR TITLE
feature: full text search for the clients endpoint

### PR DESCRIPTION
**Pull request by rtirserio**
Enables full text search for clients endpoint

### CODE REVIEW

File: src/services/employees.js [hunk 1]
```diff
@@ -77,24 +82,44 @@ const get_clients_filters = (params = {}) => {
...
     qb.join('roles', 'employees.role', 'roles.code')
     qb.leftOuterJoin('employees as manager', 'employees.manager_id', 'manager.perdet_seq_num')
-    qb.where(query)
+    let q = params
+    if (mapping) {
+      q = mapping(params)
+    }
+    qb.where(q)
+
+    addFreeTextFilter(qb, params['request_params.fullText'])
```
Comment (scott062) [inline src/services/employees.js]: is it '.fullText' or '.freeText'?

Comment (rtirserio) [reply src/services/employees.js]: The param name for the PV/AP search is `freeText` so its possible that I messed that up

Comment (scott062) [reply src/services/employees.js]: /clients endpoint is still work in progress with web service team, but swagger params show it as .freeText, so I'd expect that for now.

Comment (rtirserio) [reply src/services/employees.js]: @scott062 Fixed the parameter

**Pull request merged**